src/backends/plotly.jl

```diff
@@ -432,7 +432,7 @@ function plotly_series(plt::Plot, series::Series)
 
     # for surface types, set the data
     if st in (:heatmap, :contour, :surface, :wireframe)
-        for letter in [:x,:y,:z]
+        for letter in [:x,:y,:z,:surfacecolor]
```

Comment:
This bit of code is for the data series (x, y and z), not for passing keyword arguments.